README.md

```diff
@@ -27,6 +27,9 @@ Demonstrate calling the method with an argument of "young prince".
 Write your code here:
 ```ruby
 # code here
+def offer_rose (person)
```

Comment:
:+1: Only suggestion would be to remove the space between the method name and parens, to better follow common ruby style.